Packs/Code42/Integrations/Code42/Code42.py

```diff
@@ -705,13 +757,27 @@ def securitydata_search_command(client, args):
             code42_security_data_context,
             headers=SECURITY_EVENT_HEADERS,
         )
-        security_data_context_key = "Code42.SecurityData(val.EventID && val.EventID == obj.EventID)"
-        context = {security_data_context_key: code42_security_data_context, "File": file_context}
-        return readable_outputs, context, file_events
+        code42_results = CommandResults(
+            outputs_prefix="Code42.SecurityData",
+            outputs_key_field="EventID",
+            outputs=code42_security_data_context,
+            readable_output=readable_outputs,
+            raw_response=file_events
+        )
+        file_results = CommandResults(
+            outputs_prefix="File",
+            outputs_key_field=None,
+            outputs=file_context,
+        )
+        return code42_results, file_results
+
     else:
         return "No results found", {}, {}
```

Comment:
else block returns tuple with 3 values whereas the earlier block above, line 772, only returns 2 values

Reply:
Thanks, I missed that